docs/dictionary/command/flip.lcdoc

```diff
@@ -2,10 +2,10 @@ Name: flip
 
 Type: command
 
-Syntax: flip [<image>] {horizontal | vertical}
+Syntax: flip [image] {horizontal | vertical}
```

Comment:
There does need to be a parameter in angle brackets here, but obviously we can't use <image> as it's a another entry. Also `flip` is a bit idiosyncratic as you must include the word `image`in the image reference, it's not a completely general one. So I suggest something like `Syntax: flip [image <imageRef>] {horizontal | vertical}` and then documenting it as
```
Parameters:
imageRef: An image name, number or id.
```